opensuse_leap/15.0/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM opensuse/leap:15.0
```

Comment:
Regarding the file path: I propose `opensuse/leap/15.0/Dockerfile` (and the tag will be `packpack/packpack:opensuse-leap-15.0`)

This would be more in-row with other names. See:

* `debian/stretch` → `packpack/packpack:debian-stretch`
* `centos/7` → `packpack/packpack:centos-7`

Reply:
Changed as suggested.